### DISCUSSION TITLE
How to output same audio to two different BT sinks simultaneously?

### DISCUSSION THREAD

**Discussion by Buttmunky1** [pschatzmann/ESP32-A2DP#604] [Q&A] [answered]
Hi, sorry back again with another question, this time wondering if it's possible to output the exact same audio simultaneously to two different sinks. At the moment, only the second-in-code speaker acknowledges being connected and plays the audio:

```
void setup() {
  // Start the bluetooth
  a2dp_source.start("Speaker 1");
  a2dp_source.start("Speaker 2");
}

void loop() {
  if (some_logic) {
    a2dp_source.start("Speaker 1", get_sound_data);
    a2dp_source.start("Speaker 2", get_sound_data);
}
```

My guess is that I am using the a2dp_source.start() method incorrectly here, and it's being immediately overwritten by the following statement? The weird part is that even if _Speaker 2_ is the one connected, if I refer to _Speaker 1_ in other methods, _Speaker 2_ still plays the audio with no issue.

**Comment by pschatzmann** [comment, marked as answer]
no
